alembic/versions/2024_01_19_bebe0db0c8de_rename_sars_cov_2_to_mutant.py

```diff
@@ -0,0 +1,44 @@
+"""rename_sars-cov-2_to_mutant
+
+Revision ID: bebe0db0c8de
+Revises: 6c3da78d1af0
+Create Date: 2024-01-19 11:36:22.328953
+
+"""
+from sqlalchemy import orm
+
+from trailblazer.store.models import Analysis
+
+# revision identifiers, used by Alembic.
+revision = "bebe0db0c8de"
+down_revision = "6c3da78d1af0"
+branch_labels = None
+depends_on = None
+
+import sqlalchemy as sa
+
+from alembic import op
+
+
+def upgrade():
+    bind = op.get_bind()
+    session = orm.Session(bind=bind)
+
+    for analysis in session.query(Analysis).filter(Analysis.data_analysis == "sars-cov-2"):
+        print(f"Altering analysis: {str(analysis)}")
```

Comment:
Do we need the prints in the alembic script, or was it mainly for debugging?
```suggestion

```

Reply:
No, just debugging and not very useful. Will remove.